content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -186,6 +186,126 @@ Resource associated with Activity:
 This output has both the span created to track work in the route, and an
 automatically-created span that tracks the inbound ASP.NET Core request itself.
 
+## Send traces to a Collector
+
+The [OpenTelemetry Collector](/docs/collector/getting-started/) is a vital
+component of most production deployments. A collector is most beneficial in the following situations, among others:
+
+* A single telemetry sink shared by multiple services, to reduce overhead of
+  switching exporters
+* Aggregate traces across multiple services, running on multiple hosts
+* A central place to process traces prior to exporting them to a backend
+
+### Configure and run a local collector
```

Comment:
```suggestion
### Configure and run a local Collector
```

Reply:
Or change Ln189 to use lowercase `collector`

Reply:
Should be lower case.